CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributing to the Cordova Showcase Application Template
+
+The Cordova Showcase Application Template is part of the [AeroGear project](https://aerogear.org/), see the [Community Page](https://aerogear.org/community) for general guidelines for contributing to the project.
+
+This document details specifics for contributions to the Cordova Showcase Application Template.
+
+## Issue tracker
+
+The tracking of issues for the AeroGear Cordova Showcase Application Template is done in the [AeroGear Android Project](https://issues.jboss.org/projects/AEROGEAR/issues) in the [JBoss Developer JIRA](https://issues.jboss.org).
+
+See the [AeroGear JIRA Usage and Guidelines Guide](https://aerogear.org/docs/guides/JIRAUsage/) for information on how the issue tracker relates to contributions to this project.
+
+## Asking for help
+
+Whether you're contributing a new feature or bug fix, or simply submitting a
+ticket, the Aerogear team is available for technical advice or feedback. 
+You can reach us at [#aerogear](ircs://chat.freenode.net:6697/aerogear) on [Freenode IRC](https://freenode.net/) or the 
+[aerogear google group](https://groups.google.com/forum/#!forum/aerogear)
+-- both are actively monitored.
+
+## Prerequisites
+
+Ensure you have the following installed in your machine:
+
+- [Node](https://nodejs.org/en/) - version 8.9.4
+- [Ionic](https://ionicframework.com/) - version 3.20.0
+- [Cordova](https://cordova.apache.org/) - version 8.0.0
+
+
+# Developing the Cordova Showcase Application Template for testing new SDK features
+This application is intended to showcase the latest release of [Aerogear Js SDK](https://github.com/aerogear/aerogear-js-sdk), so pre-release work should never be merged to the `master` branch, any changes made pre-release can be merged to the `development` branch only.
+
+## Cloning the repository
+You will first need to clone [Aerogear Js SDK](https://github.com/aerogear/aerogear-js-sdk):
+```
+git clone git@github.com:aerogear/aerogear-js-sdk.git
+```
+Next, clone the showcase app directly into the SDK:
+```
+cd aerogear-js-sdk
+git clone git@github.com:aerogear/cordova-showcase-template.git cordova-example
+```
+## Installing dependencies
+As we are developing and testing newly implemented features and changes we need to point our dependencies in the example app `package.json` to local versions of `core` and `auth`:
+
+```
+"dependencies": {		    
+    "@aerogear/auth": "../packages/auth",		 
```

Comment:
Preferred options is to run `npm run bootstrap` that will link the packages instead of installing them from NPM. The app folder has to be added in `lerna.json` though.

Reply:
@josemigallas mind having another look, I added changes to the build steps on using `npm run bootstrap` in this commit https://github.com/aerogear/cordova-showcase-template/pull/13/commits/463dcbd4f0f7b73ca66523cfbb381052d0f4f978
I had to point the dependencies to the local versions in order to build and run the app with no errors.